app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -486,7 +486,8 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler,
                     sessionId = customTabSessionId,
                     fragmentManager = parentFragmentManager,
                     launchInApp = { context.settings().openLinksInExternalApp },
-                    loadUrlUseCase = context.components.useCases.sessionUseCases.loadUrl
+                    loadUrlUseCase = context.components.useCases.sessionUseCases.loadUrl,
+                    failedToLaunchAction = { context.components.useCases.sessionUseCases.loadUrl }
```

Comment:
You are not actually invoking the use case in the lambda? 🤔

Reply:
Oh, let me double check.  I tested this and it works as I was expecting but it could be that my build wasn't clean and had other changes.

Reply:
This works but I think you're right that this is not the best.  After looking into this I think I'll make a change in A-C first to make this much cleaner.  Thanks,

Reply:
Once this is in https://github.com/mozilla-mobile/android-components/pull/9076. I'll update this to use the new boolean.